handbook/company/pricing-features-table.yml

```diff
@@ -98,6 +98,7 @@
   tier: Free
   jamfProHasFeature: yes
   jamfProtectHasFeature: no
+  isExperimental: yes
```

Comment:
```suggestion
```

I think not experimental. macOS and Windows are shipped and stable. iOS isn't experimental yet b/c it's not shipped.